lib/router.js

```diff
@@ -69,7 +74,7 @@ Cherrytree.prototype.map = function (routes) {
 
   eachBranch({routes: this.routes}, [], function (routes) {
     // concatenate the paths of the list of routes
-    var path = _.reduce(routes, function (memo, r) {
+    var path = routes.reduce(function (memo, r) {
```

Comment:
hm, why did you switch this up?

I _think_ I'm using my own _. implementations to support non es5 environment. Not sure it's worth it though, since you can polyfill that. Perhaps I should switch it all to using es5 array methods and similar. Would reduce the size of the lib and simplify the code.